Dockerfile

```diff
@@ -177,39 +180,38 @@ ENV PATH $PATH:/app/node_modules/.bin
 ENV NODE_ENV production
 
 # Copy the configuration files to the apps/project root
-COPY --from=builder --chown=appuser:appuser /app/apps/${PROJECT}/next.config.js \
+COPY --from=builder /app/apps/${PROJECT}/next.config.js \
     /app/apps/${PROJECT}/i18nRoutes.config.js \
     /app/apps/${PROJECT}/next-i18next.config.js \
     /app/apps/${PROJECT}/package.json \
     ./apps/${PROJECT}/
 
 # Copy common-i18n to share localization files after build
-COPY --from=builder --chown=appuser:appuser /app/packages/common-i18n/src/locales \
+COPY --from=builder /app/packages/common-i18n/src/locales \
     ./packages/common-i18n/src/locales
 
 # Automatically leverage output traces to reduce image size
 # https://nextjs.org/docs/advanced-features/output-file-tracing
 # copy only the necessary files for a production deployment including select files in node_modules.
-COPY --from=builder --chown=appuser:appuser /app/apps/${PROJECT}/.next/standalone ./apps/${PROJECT}/
-COPY --from=builder --chown=appuser:appuser /app/apps/${PROJECT}/.next/static ./apps/${PROJECT}/.next/static
-COPY --from=builder --chown=appuser:appuser /app/apps/${PROJECT}/public ./apps/${PROJECT}/public
+COPY --from=builder /app/apps/${PROJECT}/.next/standalone/apps/${PROJECT} ./apps/${PROJECT}/
+COPY --from=builder /app/apps/${PROJECT}/.next/static ./apps/${PROJECT}/.next/static
+COPY --from=builder /app/apps/${PROJECT}/public ./apps/${PROJECT}/public
 
-# The root level files
-COPY --from=builder --chown=appuser:appuser /app/node_modules ./node_modules
-COPY --from=builder --chown=appuser:appuser /app/package.json ./package.json
+# Copy the *modified* node_modules from the build stage
+COPY --from=builder /app/apps/${PROJECT}/.next/standalone/node_modules ./apps/${PROJECT}/node_modules
 
 # OpenShift write access to Next cache folder
-USER root
-RUN chgrp -R 0 /app/apps/${PROJECT}/.next/server/pages && chmod g+w -R /app/apps/${PROJECT}/.next/server/pages
-USER appuser
+#USER root
+#RUN chgrp -R 0 /app/apps/${PROJECT}/.next/server/pages && chmod g+w -R /app/apps/${PROJECT}/.next/server/pages
+#USER appuser
```

Comment:
I'm fixing this now, so this is just temporary.

Reply:
👍